config/settings/test.py

```diff
@@ -1,4 +1,20 @@
-from .dev import *  # noqa
+from .base import *  # noqa: F401
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "@6gce61jt^(pyj5+l**&*_#zyxfj5v1*71cs5yoetg-!fsz826"
+
+# SECURITY WARNING: define the correct hosts in production!
+ALLOWED_HOSTS = ["*"]
+
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+try:
+    from .local import *  # noqa: F401
+except ImportError:
+    pass
 
 BIRDBATH_REQUIRED = False
```

Comment:
We should probably override the password hasher too, as advised in the Django docs: https://docs.djangoproject.com/en/3.2/topics/testing/overview/#password-hashing

Reply:
The marked code block was previously part of <from .dev import * # noqa>.  For Password hasher,  not authenticating many users. Have overridden password hasher. Nor sure what to do about the selected code block?